src/helper-functions/search-members.js

```diff
@@ -0,0 +1,11 @@
+const searchMembers = (membersArr, searchTerm) => {
+  return membersArr.filter(
+    (member) =>
+      member.first_name
+        .toLowerCase()
+        .includes(searchTerm.toLocaleLowerCase()) ||
+      member.last_name.toLowerCase().includes(searchTerm.toLocaleLowerCase())
+  );
+};
```

Comment:
How about using `.find()` method?

Reply:
find() will look and stop after the first match, whereas, .filter() will continue searching through the entire array